stl/inc/expected

```diff
@@ -0,0 +1,1035 @@
+// expected standard header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _EXPECTED_
+#define _EXPECTED_
+#include <yvals.h>
+#if _STL_COMPILER_PREPROCESSOR
+#ifndef __cpp_lib_expected
+#pragma message("The contents of <expected> are available only with C++23 or later.")
+#else // ^^^ !__cpp_lib_expected / __cpp_lib_expected vvv
+#include <exception>
+#include <initializer_list>
+#include <type_traits>
+#include <xutility>
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+
+template <class _Err>
+class unexpected;
+
+template <class _Err>
+struct _Check_unexpected_argument : true_type {
+    static_assert(is_object_v<_Err>, "E must be an object type. (N4910 [expected.un.object.general]/1)");
+    static_assert(!is_array_v<_Err>, "E must not be an array type. (N4910 [expected.un.object.general]/1)");
+    static_assert(!is_const_v<_Err>, "E must not be const. (N4910 [expected.un.object.general]/1)");
+    static_assert(!is_volatile_v<_Err>, "E must not be volatile. (N4910 [expected.un.object.general]/1)");
+    static_assert(!_Is_specialization_v<_Err, unexpected>,
+        "E must not be a specialization of unexpected. (N4910 [expected.un.object.general]/1)");
+};
+
+// [expected.un.object]

```

Comment:
Actually I do see this stable name:
<img width="773" alt="stable_name" src="https://user-images.githubusercontent.com/4231088/169748358-e74c6cf5-d510-46fd-8cf8-029ba38ca2c6.png">
That said, I have no objection to the change you made here.

Reply:
These two subclauses are one in my copy; I assume they were merged editorially, since it's a common occurrence for one-paragraph subclauses. Yes, that's it: https://github.com/cplusplus/draft/pull/5360.